systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -553,7 +553,7 @@ private void afterAllMustExecute(ExtensionContext extensionContext)  {
                 LOGGER.debug("{} - Configurations of previous Cluster Operator are not identical. Starting rollback to the default configuration.", extensionContext.getRequiredTestClass().getSimpleName());
                 LOGGER.debug("Current Cluster Operator configuration:\n" + clusterOperator.toString());
                 LOGGER.debug("Default Cluster Operator configuration:\n" + clusterOperator.defaultInstallation().createInstallation().toString());
-                LOGGER.info("Current Cluster Operator configuration differs from default Cluster Operator in these attributes:{}", clusterOperator.diff(clusterOperator.defaultInstallation().createInstallation()));
+                LOGGER.info("Current Cluster Operator configuration differs from default Cluster Operator in these attributes: {}", clusterOperator.diff(clusterOperator.defaultInstallation().createInstallation()));
```

Comment:
CO should be together or not?

Reply:
I'd refference to openshift docs where Cluster Operator is separate, or would you like it to be together?